bin/kano-feedback-widget-cli

```diff
@@ -12,24 +12,39 @@
 """Kano Feedback Widget CLI
 
 Usage:
-  kano-feedback-widget-cli <qid> <response>
+  kano-feedback-widget-cli [--dry-run] [--debug] (<question_id> <response>)...
 
 Options:
-  -h --help     Show this screen.
+  --question=<response>    String in the form "qid:response", multiple questions accepted
```

Comment:
Just for the sake of consistency, this line which isn't needed anymore should be removed.